nemo/utils/exceptions.py

```diff
@@ -35,3 +35,13 @@ def __call__(self, *args, **kwargs):
 
     def __getattr__(self, *args, **kwargs):
         raise LightningNotInstalledException(self)
+
+
+class LossMisconfigurationException(NeMoBaseException):
+    def __init__(self, obj):
+        message = (
+            f"Error while computing the loss in {obj}. Make sure your loss parameters are configured "
+            "properly to match your dataset setup (ex. making sure `train_ds.min_duration` is long "
+            "enough for `ContrastiveLoss.num_negatives`.)"
```

Comment:
Perhaps move this text to only the contrastive loss instance of this exception?

Reply:
Sure, can add a separate `msg` arg where someone can provide a custom message

Reply:
@piraka9011 Could you please update the PR with the change requested above? Thanks!

Reply:
Sorry! Missed this. See latest changes.